client/status.v2/status_interactive.go

```diff
@@ -205,6 +205,10 @@ func interactive(c Client, flag statusFlags) error {
 				})
 			}
 		}()
+		if flag.Delay < time.Second*1 {
+			// minimum delay 1 sec to avoid high cpu usage
```

Comment:
I think the 1 second is setup-dependent. FWIW my laptop setups is just fine with an interval of 300ms. Regardless, I am strongly silently adjusting the value here.

Reply:
> This looks great!

Thank you

> 
>     * [ ]  The implicit delay-adjustment should go away IMHO. As long as we keep the default at 1s I don't see why we shouldn't allow users to set delay freely, they can't do much harm.

ok I understand and that is not a problem, I just removed the delay-adjustment.

> 
>     * [ ]  There needs to be an entry in `changelog.rst` for the renamed signal.

I added a simple entry in the `changelog.rst` with, I guess the 0.4.0 version, but not sure whether that is ok, I considered the rename as a break change because any script using the wakeup signal will need to be changed with the new sintax. This may need more information that I would prefer you to update.

> 
> 
> Are you planning more changes?

no, nothing else at the moment